Project.toml

```diff
@@ -42,6 +42,15 @@ Triangulate = "f7e6ffb2-c36d-4f8f-a77e-16e897189344"
 TriplotBase = "981d1d27-644d-49a2-9326-4793e63143c3"
 TriplotRecipes = "808ab39a-a642-4abf-81ff-4cb34ebbffa3"
 
+[weakdeps]
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
+
+[extensions]
+TrixiMakieExt = "Makie"
+
+[extras]
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
```

Comment:
I think the usual ordering (imposed by Aqua.jl) is to have `[extras]` after `[compat]`

Reply:
OK, we (I) can change that when I'm back at a keyboard.

Reply:
Done in 52ddac3.